idea_town/experiments/views.py

```diff
@@ -14,7 +14,7 @@ class ExperimentViewSet(viewsets.ModelViewSet):
     Returns a list of all Experiments in the system.
     """
     queryset = Experiment.objects.all()
-    serializer_class = ExperimentSerializer
+    serializer_class = ExperimentDeepSerializer
```

Comment:
FWIW, the shallow `ExperimentSerializer` is used here to try to keep DB queries & response sizes down in case we have a large number of `Experiments` and don't need **all** the `ExperimentDetail`  for list views. Do we need to tweak the shallow serializer rather than just use the deep one?